aptos-move/move-examples/messageboard/sources/Offer.move

```diff
@@ -0,0 +1,138 @@
+/// Provides a way to transfer structs from one account to another in two transactions.
```

Comment:
I feel like this should just be part of the standard library if you want to incorporate it.

Reply:
added to stdlib